js/cms.js

```diff
@@ -1,13 +1,30 @@
-// Fetch page data from Strapi
-function getPageData(page, source) {
-	const cms = source;
-	let apiURL = "";
-
-	if (cms === "strapi") {
-		apiURL = `https://cms.iftheselandscouldtalk.org/api/${page}?populate=*`;
-	} else if (cms === "wordpress") {
-		apiURL = `https://wp.iftheselandscouldtalk.org/wp-json/wp/v2/page`;
+// Helper functions
+function appendData(data, element, method, target) {
```

Comment:
This works, but it's a little brittle right now. I need to add some checks to make sure the targets exist, etc.